// d3-array/index.js
export { default as bisect, bisectRight, bisectLeft } from "./src/bisect.js";
export { default as ascending } from "./src/ascending.js";
export { default as bisector } from "./src/bisector.js";
export { default as descending } from "./src/descending.js";
export { default as deviation } from "./src/deviation.js";
export { default as extent } from "./src/extent.js";
export { default as histogram } from "./src/histogram.js";
export { default as thresholdFreedmanDiaconis } from "./src/threshold/freedmanDiaconis.js";
export { default as thresholdScott } from "./src/threshold/scott.js";
export { default as thresholdSturges } from "./src/threshold/sturges.js";
export { default as max } from "./src/max.js";
export { default as mean } from "./src/mean.js";
export { default as median } from "./src/median.js";
export { default as merge } from "./src/merge.js";
export { default as min } from "./src/min.js";
export { default as pairs } from "./src/pairs.js";
export { default as permute } from "./src/permute.js";
export { default as quantile } from "./src/quantile.js";
export { default as range } from "./src/range.js";
export { default as scan } from "./src/scan.js";
export { default as shuffle } from "./src/shuffle.js";
export { default as sum } from "./src/sum.js";
export { default as ticks, tickStep } from "./src/ticks.js";
export { default as transpose } from "./src/transpose.js";
export { default as variance } from "./src/variance.js";
export { default as zip } from "./src/zip.js";
